src/sessions/session-manager.ts

```diff
@@ -159,6 +174,12 @@ export class SessionManager {
    * @param sessionInfo pending webrtc-session info
    */
   async onPropose (sessionInfo: ISessionInfo): Promise<void> {
+    const handler = this.getSessionHandler({ sessionInfo, allowUnknownHandler: true });
+
+    if (handler.disabled) {
+      return;
```

Comment:
The noise is why I'm not logging this. I didn't want to create red herrings in the logs because, at least for softphone, proposes get sent out every 1s and I didn't want the logs to contain a bunch of "Ignoring propose for ____". I was afraid of getting a bunch of q's about why calls were being ignored when they really weren't (just for this client). This will be especially valid when the volt feature toggle gets turned on because then this would happen for *every* call received by web dir.